App.tsx

```diff
@@ -17,33 +17,102 @@ import {
   Text,
   StatusBar,
   Button,
-} from 'react-native';
+  Keyboard,
+} from "react-native";
+import { TouchableWithoutFeedback } from "react-native";
 
 import {
   Header,
   LearnMoreLinks,
   Colors,
   DebugInstructions,
   ReloadInstructions,
-} from 'react-native/Libraries/NewAppScreen';
-import WelcomeScreen from './WelcomeScreen';
+} from "react-native/Libraries/NewAppScreen";
+import WelcomeScreen from "./WelcomeScreen";
+import SignUp from "./SignUp";
 
-declare var global: {HermesInternal: null | {}};
+declare var global: { HermesInternal: null | {} };
 
 const App = () => {
   return (
-    <>
-      <WelcomeScreen />
```

Comment:
We should not remove the Welcome Screen. We want to display the WelcomeScreen upon startup. And the user will be shown the registration page upon clicking on the 'Get Started' button. Please, make the appropriate changes.

Reply:
ok